packages/common/env/package.json

```diff
@@ -7,8 +7,7 @@
     "@blocksuite/store": "0.11.0-nightly-202401020419-752a5b8",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "vitest": "1.1.3",
-    "zod": "^3.22.4"
```

Comment:
This was being used at runtime even though it was specified as a dev dependency.